egs/wsj/s5/steps/segmentation/convert_targets_dir_to_whole.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+# Copyright 2017  Vimal Manohar
+# Apache 2.0
+
+nj=4
+cmd=run.pl
+default_targets=   # vector of default targets in text format
+
+set -o pipefail -u
+
+[ -f ./path.sh ] && . ./path.sh
+. utils/parse_options.sh
+
+if [ $# -ne 4 ]; then
+  cat <<EOF
+  Usage: steps/segmentation/convert_targets_to_whole.sh <data-dir> <whole-data-dir> <targets-dir> <whole-targets-dir>
+   e.g.: steps/segmentation/convert_targets_to_whole.sh \
```

Comment:
I would add a descriprion as to what the utility is for. The concept of "whole" in this context also probably needs a little elaboration.